packages/@react-spectrum/illustratedmessage/docs/IllustratedMessage.mdx

```diff
@@ -36,7 +36,7 @@ category: Content
 
 ```tsx example
 <IllustratedMessage>
-  <svg width="150" height="103" viewBox="0 0 150 103" aria-hidden="true" role="presentation">
+  <svg width="150" height="103" viewBox="0 0 150 103" role="presentation">
```

Comment:
Should be the other way. `role="presentation"` only hides the svg element, not its children.`aria-hidden` hides all descendants. But, shouldn't be needed after the illustrations go in.